src/components/Login/Login.js

```diff
@@ -32,7 +31,12 @@ const Login = () => {
   };
 
   return (
-    <div className="login">
+    <div
+      className="login"
+      onClick={e => {
+        e.target.className === 'login' && setLoginWindowBtn(false);
+      }}
+    >
```

Comment:
```suggestion
     <div className="login" onClick={handleModal}>
```
제가 파악했을 때 이 부분은 모달창 바깥 부분을 클릭하면
모달창이 닫히는 동작을 구현하기 위해 작성하신 코드로 보입니다.

제가 수정한대로 구현한다면 입력을 위해 인풋 창을 클릭하면 똑같이 모달창이 닫히게 되는데
그 때는 바로 아래 있는 div  요소에 e.stopPropagation()를 onclick 이벤트로 걸어주시면 해결이 됩니다!

Reply:
아아 그렇게 하면 되네요! 이해했습니다 감사합니다!!